src/tap_server/__init__.py

```diff
@@ -0,0 +1,11 @@
+"""
+TAP Server Prototype
+
+A prototype implementation of a TAP (Table Access Protocol) server
+following the IVOA TAP specification v1.1.
```

Comment:
```suggestion
following the IVOA TAP specification v1.1 (sync-only implementation).
```
Could clarify we're not currently interested in implementing all TAP 1.1 endpoints